src/lib/components/BucketAggregation/BucketAggregationValues.js

```diff
@@ -97,11 +100,13 @@ BucketAggregationValues.propTypes = {
   }),
   onFilterClicked: PropTypes.func.isRequired,
   overridableId: PropTypes.string,
+  parentValue: PropTypes.string,
```

Comment:
```suggestion
  parentValue: PropTypes.string.isRequired,
```

Reply:
We are going for a different solution, where we don't specify any more the id of the element. There is no need to pass the parent anymore